### PR TITLE
fix: set ws.Server maxPayload to prevent oom

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

File: src/lib/app.js [hunk 1]
```diff
@@ -55,6 +56,16 @@ class App {
     koaApp.ws.use(passport.initialize())
     koaApp.ws.use(websocketRouter.routes())
     koaApp.ws.use(websocketRouter.allowedMethods())
+    // Override listen() in order to specify a maxPayload. See:
+    //   https://github.com/kudos/koa-websocket/blob/785001c20d94fbb60cb05596bb1869889f539351/index.js#L15-L20
+    // for the original version.
+    koaApp.ws.listen = function (server) {
```
Comment (emschwartz) [inline src/lib/app.js]: If you need to overwrite one of 3 methods provided by this library is it even worth using the library? If we do stick with this we should submit a PR or file an issue on the module's repo about allowing you to specify additional options.

File: src/lib/app.js [hunk 2]
```diff
@@ -55,6 +56,16 @@ class App {
...
+      this.server = new WebSocketServer({
+        server: server,
+        maxPayload: 64 * 1024
```
Comment (emschwartz) [inline src/lib/app.js]: It would be good to put this as a constant up at the top of the file and provide a justification for the size chosen

**Comment by sentientwaffle** [comment]
@emschwartz ok, fixed

File: src/lib/koa-websocket.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+'use strict'
```
Comment (sentientwaffle) [inline src/lib/koa-websocket.js]: Ok. There's already a month-old issue at https://github.com/kudos/koa-websocket/issues/22.

Comment (emschwartz) [inline src/lib/koa-websocket.js]: Gotcha. In that case, sounds fine to copy the code locally and remove the dependency on that package

**Pull request merged**